db/re/skill_require_db.txt

```diff
@@ -1054,7 +1054,7 @@
 
 // New Arch Bishop Skills
 5072,0,0,144:120:106:92:78,0,0,0,99,0,0,none,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0				//AB_VITUPERATUM##
-5073,0,0,70,0,0,0,99,0,0,none,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0								//AB_CONVENIO##
+5073,0,0,70,0,0,0,99,0,0,none,0,0,717,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0								//AB_CONVENIO##
```

Comment:
Skill 5073 doesn't need item 717.
Skill 5072 needed.
info : https://www.divine-pride.net/database/skill/5072

Reply:
I tested skill AB_CONVENIO always warp partymember to wrong map